### ISSUE TITLE
No input_files in assemblytainer

### ISSUE THREAD

**Issue by jansule** [o2r-project/o2r-meta#31]
No input_files will be extracted after uploading the assemblytainer (stored in o2r-data >Hilfskräfte > Suleiman). @7048730 mentioned that this works already, but it does not (on my machine).

**Comment by 7048730** [comment]
With "assemblytainer" you mean "AssemblyProcessesOfGastropodCommunityChange" in RR papers?

**Comment by 7048730** [comment]
Here is the content of `inputfiles` when I extract from that paper. It seems to work. Are you using the latest version?

```json
    "inputfiles": [
        "extract/tests/all/AssemblyProcessesOfGastropodCommunityChange/AssemblyProcessesOfGastropodCommunityChange/Supplement/OhridComm.csv",
        "extract/tests/all/AssemblyProcessesOfGastropodCommunityChange/AssemblyProcessesOfGastropodCommunityChange/Supplement/TaxDistance.csv",
        "extract/tests/all/AssemblyProcessesOfGastropodCommunityChange/AssemblyProcessesOfGastropodCommunityChange/Supplement/Localities.csv",
        "extract/tests/all/AssemblyProcessesOfGastropodCommunityChange/AssemblyProcessesOfGastropodCommunityChange/Supplement/StepcamResultHauffe.csv",
        "extract/tests/all/AssemblyProcessesOfGastropodCommunityChange/AssemblyProcessesOfGastropodCommunityChange/Supplement/OhridComm.csv",
        "extract/tests/all/AssemblyProcessesOfGastropodCommunityChange/AssemblyProcessesOfGastropodCommunityChange/Supplement/TaxDistance.csv",
        "extract/tests/all/AssemblyProcessesOfGastropodCommunityChange/AssemblyProcessesOfGastropodCommunityChange/Supplement/Localities.csv",
        "extract/tests/all/AssemblyProcessesOfGastropodCommunityChange/AssemblyProcessesOfGastropodCommunityChange/Supplement/OhridComm.csv",
        "extract/tests/all/AssemblyProcessesOfGastropodCommunityChange/AssemblyProcessesOfGastropodCommunityChange/Supplement/TaxDistance.csv",
        "extract/tests/all/AssemblyProcessesOfGastropodCommunityChange/AssemblyProcessesOfGastropodCommunityChange/Supplement/Localities.csv",
        "extract/tests/all/AssemblyProcessesOfGastropodCommunityChange/AssemblyProcessesOfGastropodCommunityChange/Supplement/StepcamResultHauffe.csv"
    ]
```

**Comment by jansule** [comment]
I just pulled the latest version of muncher. Now, I get some inputfiles. In my case, only 7 files are found. Here is my content of `inputfiles`:
```JSON
"inputfiles": [
                "/tmp/o2r/compendium/lVAo0/data/Supplement/OhridComm.csv",
                "/tmp/o2r/compendium/lVAo0/data/Supplement/TaxDistance.csv",
                "/tmp/o2r/compendium/lVAo0/data/Supplement/Localities.csv",
                "/tmp/o2r/compendium/lVAo0/data/Supplement/OhridComm.csv",
                "/tmp/o2r/compendium/lVAo0/data/Supplement/TaxDistance.csv",
                "/tmp/o2r/compendium/lVAo0/data/Supplement/Localities.csv",
                "/tmp/o2r/compendium/lVAo0/data/Supplement/StepcamResultHauffe.csv"
            ]
```

Is there a reason some files have duplicates?

**Comment by 7048730** [comment]
Thanks for the hint. which ERC did you use?

**Comment by nuest** [comment]
The title of the issue mentions _assemblytainer_.

**Comment by 7048730** [comment]
silly me.
The dublicates are already in my own example. They are the result of parsing the rmd of the assemblytainer that does indeed load the files multiple times. So I will uniqueness filter to the listing function.

**Comment by 7048730** [comment]
fixed with 3f5d7feeda907975586df4adc63cbbe3da34d2d8
